dataframe-csv/src/test/resources/abc.tsv

```diff
@@ -0,0 +1,3 @@
+A	B	C
```

Comment:
Maybe it'd be ok to just read files from `../core/src/test/resources`?

Reply:
maybe yeah, but I think once we remove the old csv implementation from :core we can better remove the tests and resources there. Then they're not duplicate anymore